ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign-v4-simulation/permit-simulation/permit-simulation.tsx

```diff
@@ -97,19 +99,29 @@ const PermitSimulation: React.FC<object> = () => {
             value={message.value}
             tokenId={message.tokenId}
             chainId={chainId}
+            message={message}
+            canDisplayValueAsUnlimited
           />
         )}
       </Box>
     </ConfirmInfoRow>
   );
 
+  let descriptionKey;
+  if (isNFT) {
+    descriptionKey = 'simulationDetailsApproveDesc';
+  } else if (message.allowed === false) {
+    // revoke permit
+    descriptionKey = 'revokeSimulationDetailsDesc';
+  } else {
```

Comment:
Minor, rather than setting the fallback with an `else`, could we assign it by default when we define `descriptionKey`?